File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	List<TbBrand> findAll();
	
	PageResult findPage(Integer pageNum,Integer pageSize);
	
	void add(TbBrand brand);
	
	TbBrand findById(Long id);
	
	void update(TbBrand brand);
	
	void delete(Long[] ids);
	
	PageResult findPage(TbBrand brand,Integer pageNum,Integer pageSize);
	
	List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function($http){
	this.findAll=function(){
		return $http.get("../brand/findAll.do");
	} 
	this.findPage=function(page,size){
		return $http.get("../brand/findPage.do?page="+page +"&size="+size);
	}
	this.add=function(entity){
		return $http.post("../brand/add.do",entity);
	}
	this.update=function(entity){
		return $http.post("../brand/update.do",entity);
	}
	this.findOne=function(id){
		return $http.get("../brand/findById.do?id="+id);
	}
	this.delSelect=function(ids){
		return $http.get("../brand/delete.do?ids="+ids)
	}
	this.search=function(page,size,brandEntity){
		return $http.post("../brand/search.do?page="+page +"&size="+size,brandEntity)
	}
	this.selectOptionList=function(){
		return $http.get("../brand/selectOptionList.do")
	}
})